tests/scripts/all.sh

```diff
@@ -1125,6 +1125,15 @@ component_build_armcc () {
     armc6_build_test "--target=aarch64-arm-none-eabi -march=armv8.2-a"
 }
 
+component_build_armcc_uecc_baremetal () {
+    msg"build: ARM Compiler 5, make with uecc and baremetal"
```

Comment:
Typo: missing space between `msg` and the quote.

(Note: you can test a component locally with `tests/scripts/all.sh build_armc_uecc_baremetal` - you don't have to run the whole script, which would be prohibitively long.)